### PR TITLE
[15.0][ADD] website_sale_payment

### PR DESCRIPTION
**Pull request by dsolanki-initos**
This module depends on "stock_picking_on_hold".
PR: https://github.com/OCA/sale-workflow/pull/2073

### CODE REVIEW

**Comment by rousseldenis** [comment]
@dsolanki-initos Is it a real migration or an addition ?

You should enhance the module description as it does not reflect what it does really.

Moreover it depends on website_sale module, IMHO the module name should take this into account too

**Comment by dsolanki-initos** [comment]
> @dsolanki-initos Is it a real migration or an addition ?
> 
> You should enhance the module description as it does not reflect what it does really.
> 
> Moreover it depends on website_sale module, IMHO the module name should take this into account too

Hello @rousseldenis ,
It is an addition not the migration.
I've added more description about this module. Also updated module name considering dependant module.

File: wesite_sale_payment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Website Sale Payment",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "author": "ERP Harbor Consulting Services,"
```
Comment (jans23) [inline wesite_sale_payment/__manifest__.py]: Add Nitrokey GmbH

Comment (dsolanki-initos) [reply wesite_sale_payment/__manifest__.py]: Hello @jans23 

I've already added and you can check this URL for the same. 

https://github.com/OCA/sale-workflow/blob/3a8c23e24f1cacfdae3f8c0fec8776f72f2514df/wesite_sale_payment/__manifest__.py#L5-L8

Regards,

Dhara Solanki

**Comment by dsolanki-initos** [comment]
@AaronHForgeFlow Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@MateuGForgeFlow Can you please review this PR?

**Comment by AaronHForgeFlow** [comment]
Can you update the PR title according to the new name? BTW I think the name is too generic, this modules has a very specific function.

**Comment by dsolanki-initos** [comment]
> Can you update the PR title according to the new name? BTW I think the name is too generic, this modules has a very specific function.

I've changed the PR title

**Comment by dsolanki-initos** [comment]
@CarlosRoca13 Can you please review this PR?

**Review by ikapasi-initos — APPROVED** [review]
Code Review and functional test.
LGTM

**Comment by dsolanki-initos** [comment]
@omar7r Could you please review this PR?

**Comment by CarlosRoca13** [comment]
The name of the module is so generic, and does not reflect the intentionality of the module.

In addition, it needs to be checked why the tests are not passing and the runboat is not being built. Please check this before mentioning it to all reviewers, to avoid noise in their inbox.

**Pull request closed without merging**